packages/server/postgres/utils/checkTeamEq.ts

```diff
@@ -1,16 +1,15 @@
-import Team from '../../database/types/Team'
 import getRethink from '../../database/rethinkDriver'
-import {checkTableEq} from './checkEqBase'
+import Team from '../../database/types/Team'
 import getTeamsByIds from '../queries/getTeamsByIds'
+import {checkTableEq} from './checkEqBase'
 
 const alwaysDefinedFields: (keyof Partial<Team>)[] = [
   'name',
   'createdAt',
   'isArchived',
   'isPaid',
   'tier',
-  'orgId',
-  'updatedAt'
```

Comment:
I think it'd be more clear to create another array called `ignoredFields` and use that to filter out fields we don't care.

Reply:
I fixed it, plus added somehow missing fields like 'createdBy', 'isOnboardTeam'.